packages/bruno-app/src/utils/url/index.js

```diff
@@ -18,10 +18,14 @@ export const parseQueryParams = (query) => {
     return [];
   }
 
-  let params = query.split('&').map((param) => {
-    let [name, value = ''] = param.split('=');
-    return { name, value };
-  });
+  let params = query
```

Comment:
would it make sense to use `URLSearchParams` in this case that'd follow URL Standard?

Reply:
The suggestion makes sense, I'll take a look. To consider here is that the URL field and query-params fields are updated based on each-other's value. That means during typing character by character, we may find ourselves in a state where the url is NOT correctly formatted (e.g. we only typed in half of the url envoded value so far), so the resulting string will not validate as URLSearchParams, and crash the view, unless handled with a fallback behavior which would be what is already in this PR anyway,

Reply:
Also, based on discussion in #2191, there is a design decision to make, perhaps the proper fix is even more involved (like adding extra assumptions, validations and/or UI warnings).